pkg/provisioner/reconciler/claim-reconciler/reconiler.go

```diff
@@ -265,15 +280,24 @@ func (r *ObjectBucketClaimReconciler) handleDeleteClaim(key client.ObjectKey) er
 		return nil
 	}
 	// see if obc delete is for a newly provisioned bkt vs an existing bkt
-	isDynamicallyProvisioned := isNewBucketByOB(r.internalClient, ob)
 
 	// Call the provisioner's `Revoke` method for old (brownfield) buckets regardless of reclaimPolicy.
 	// Also call `Revoke` for new buckets with a reclaimPolicy other than "Delete".
 	if ob.Spec.ReclaimPolicy == nil {
```

Comment:
de we need this check? I removed a similar check in handleProvision.

Reply:
probably a left over from a large rebase